client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -217,31 +221,37 @@ interface GetNumberColumnLabelProps {
   t: TypedTFunction<LocaleKey>;
   displayInDoses: boolean;
   displayInPack?: boolean;
+  displayDosesInUnitName?: boolean;
```

Comment:
Needed in Inbound to show unit name for a column instead of 'pack' >.>

Reply:
I don't get it... that column is just the units column? Why should it care if we are displaying another column about doses??

Reply:
Was trying to cover something... which will come back to me but will remove for now unneeded